packages/front-end/components/Settings/Team/SingleRoleSelector.tsx

```diff
@@ -57,12 +57,11 @@ export default function SingleRoleSelector({
         sort={false}
         formatOptionLabel={(value) => {
           const r = roles.find((r) => r.id === value.label);
+          if (!r) return <></>;
```

Comment:
Since the function returns a `ReactNode` it seemed preferable to just early-return an empty fragment instead of needing the expect-errors.

Reply:
In this case, I think it's better to render just the label (`value.label`) than nothing if the lookup fails.

So we can just change the code below to output `value.label` instead of `r.id` since they are identical.  And then conditionally render the description.